### PR TITLE
Math magicians app(Components)

### PR DESCRIPTION
**Pull request by edubew**
## In this milestone I did the following:
- Create a Calculator component
- Write a markup for the calculator interface
- Add basic styling to the calculator interface

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @edubew,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly use [the correct readme template with all the headers](https://github.com/microverseinc/readme-template) and ensure all the headers are properly linked

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by edubew** [comment]
> [DammyShittu](/DammyShittu)

Hello @DammyShittu, I have checked my Readme and confirmed that it is indeed the latest version. My current ReadMe file lacks just one header- FAQ, which is optional according to the template example.
I will appreciate it if you could point to exactly what I missed in my ReadMe so that I correct it. 
Thank you☺

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,10 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
```
Comment (franclobo) [inline src/App.js]: - [ ] Please refactor the App function, it should be class-based to match all the project requirements 👇🏽 

> Use class-based components (All components should be class-based).

File: src/index.css [hunk 2]
```diff
@@ -1,27 +1,45 @@
...
+.span__two {
+  grid-column: span 2;
+}
+
+.calculator__output {
+  grid-column: 1/-1;
+  background-color: gray;
+  color: #fff;
+  padding: 1rem;
+  font-size: 3rem;
```
Comment (franclobo) [inline src/index.css:40]: - [ ] For a better user interface and to be similar to the expected UI, please align the text to the right `text-align: end;`👇🏽 

| Expected | Your design |
|---|---|
| image | image |

File: README.md [hunk 3]
```diff
@@ -100,5 +99,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
-
+This project is [MIT](./MIT.md) licensed.
```
Comment (franclobo) [inline README.md]: - **[OPTIONAL]** Kindly correct the path to the LICENSE file to make sure that all the links work as expected for professional documentation 👉🏽 ` [MIT](./LICENSE)`

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @edubew, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @edubew, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- No Linters error ✅
- Followed Gitflow ✅
- Great working on the functionlaity ✅
- Professional README ✅🔥

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**